README.md

```diff
@@ -1,24 +1,175 @@
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User Registration and Authentication: Allow users to sign up for accounts and log in securely.
+- Create and Delete Transactions: Allow users to add or delete transactions to keep track of money spent
+- Add tags and icons: Provide a user-friendly option to categorise the transactions into different tags/categories
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ <!-- LIVE DEMO
+
+## 🚀 Live Demo <a name="presentation"></a>
+
+[- Live Demo - PocketCanvas](https://pocketcanvas.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Ensure you have Ruby installed on your system. You can check by running the code below in your terminal.
+```sh
+  ruby --version
+```
+### Setup
+
+Clone or download the Beyond Borders repository to your local machine.
+```sh
+  git clone https://github.com/zamcham/BlogSphere.git
+```
+Open your terminal and navigate to the cloned project directory.
+
+### Install
+
+This project does not require additional dependencies, just Ruby interpreter. However, it uses RuboCop as a linter and you should set it up as well.
+
+```sh
+bundle install
+```
+
+### Usage
+ - To run Beyond Borders, run the following command:
+
+```sh
+  rails s
+```
+Open your browser and navigate to:
+```sh
+  http://localhost:3000/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Cristian Zambrano**
+
+- GitHub: [@zamcham](https://github.com/zamcham)
+- LinkedIn: [Cristian Zambrano](https://www.linkedin.com/in/cristian-zamcham)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Edit a Category Icon**
+- [ ] **Customize your profile**
+- [ ] **See your current savings, and general balance**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions to the User Interface are welcome! If you find any issues or have suggestions for improvements, please feel free to submit a pull request or open an issue on the project repository.
+
+Feel free to check the <a href="https://github.com/christianonoh/beyond-borders/issues">issues page</a>.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give us a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Special thanks to our team and the Ruby community for their support and contributions.
+
```

Comment:
- [ ] I suggest that under your "Acknowledgements" section you give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). :pray: